File: Fibonacci/fibonacci.js
const readline = require('readline');
const rl = readline.createInterface({
    input:process.stdin,
    output:process.stdout,    
})

let cycle;
let arr = [0, 1];
let sum;


rl.question('파보나치 연산 수 입력: ', e => {
    cycle = Number(e);

    for(let i = 0; i < cycle - 1; i++) {
        arr.push(arr[i] + arr[i + 1]);
    }
    console.log(arr);

    rl.close();
})

File: Fibonacci/fibonacci.cpp
/*숫자 n이 주어졌을 때, n의
피보나치(Fibonacci) 수열을 출력하는 프로그램을 작성하시오. */
#include <iostream>

using namespace std;

int cycle;

int main () {
    cout << "파보나치 연산 횟수 입력 : ";
    cin >> cycle;

    int* arr = new int[cycle + 2];

    arr[0] = 0;
    arr[1] = 1;

    for(int i = 0; i < cycle - 1; i++) {
        arr[i + 2] = arr[i] + arr[i + 1];
    }

    for(int i=0; i < cycle + 1; i++) {
        cout << arr[i];
        if(i != cycle)
            cout << ", ";
    }
    cout << endl;

    delete[] arr;
    return 0;
}